GooglePlayInstant/Editor/QuickDeploy/AssetBundleVerifierWindow.cs

```diff
@@ -61,9 +76,57 @@ private void StartAssetBundleVerificationDownload()
 #endif
         }
 
-        private void GetAssetBundleInfoFromDownload()
+        // Visible for testing
+        internal void HandleAssetBundleVerifyState(AssetBundleVerifyState state, UnityWebRequest webRequest)
+        {
+            switch (state)
```

Comment:
I assume it's intentional that AssetBundleVerifyState.InProgress is not included here?

Reply:
Yeah. That state should never be handled, just added a clarifying comment here.